ocaml/idl/datamodel_host.ml

```diff
@@ -1766,6 +1766,13 @@ let apply_updates =
     ~allowed_roles:(_R_POOL_OP ++ _R_CLIENT_CERT)
     ()
 
+let rescan_drivers =
+  call ~name:"rescan_drivers" ~in_oss_since:None ~lifecycle:[]
+    ~doc:"Scan the host and update its driver information."
+    ~params:[(Ref _host, "host", "The host to be rescanned")]
+    ~result:(Map (String, Set String), "Updates to the drivers' versions")
```

Comment:
The return type is not obvious. The implementation defines some "keys" for what is returned, but the API documentation won't show that. I wonder how useful it is to return this data anyway; I had expected no return value. Especially since these rescans also happen implicitly on restart, where you'd never see any of it.

Reply:
I can specify the return keys in the API docs. I find it a pretty user-friendly addition, and a return value can always be ignored if the user isn't interested in it.